services/rfq/relayer/relconfig/config.go

```diff
@@ -59,6 +59,10 @@ type Config struct {
 	UseEmbeddedGuard bool `yaml:"enable_guard"`
 	// SubmitSingleQuotes enables submitting single quotes.
 	SubmitSingleQuotes bool `yaml:"submit_single_quotes"`
+	// MaxRFQSize is the amount of RFQs we are allowed to process
+	MaxRFQSize int `yaml:"rfq_size"`
```

Comment:
Is this supposed to be `MinConfirmations`? Naming is a bit confusing to me here. I would recommend a more detailed explanation (either here or in the actual `limiter.go` file) with how exactly these parameters are expected to filter a given tx